src/applications/simple-forms/40-0247/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format, isValid } from 'date-fns';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+
+export class ConfirmationPage extends React.Component {
```

Comment:
do we want to use the `/shared/components/ConfirmationPageView` for this?

Reply:
I'll look into this, and open another PR if that works out better.  Thanks!